lib/airbrakex/plug.ex

```diff
@@ -16,12 +16,27 @@ defmodule Airbrakex.Plug do
           exception ->
             session = Map.get(conn.private, :plug_session)
 
-            Airbrakex.ExceptionParser.parse(exception)
-            |> Airbrakex.Notifier.notify([params: conn.params, session: session])
+            error = Airbrakex.ExceptionParser.parse(exception)
+            if proceed?(error) do
```

Comment:
`proceed?(Application.get_env(:airbrakex, :ignore), error)` is better solution. Use pattern matching to match all cases and test this functions separately.